example/echo_client.py

```diff
@@ -599,6 +599,10 @@ def _do_closing_handshake(self):
 
 
 def main():
+    # Output in UTF-8 on Python 2. This will do the wrong thing if the
+    # locale is not UTF-8. On Python 3, the output encoding is correct
+    # for the locale but doesn't necessarily support all unicode
+    # characters. This problem is particularly prevalent on Windows.
     if six.PY2:
```

Comment:
Do we still need this?

Reply:
According to the documentation, in Python 2 the encoding defaults to "None" if the output is not a tty.

I modified it to get the same behaviour in Python 2 as in Python 3, ie. it always uses the locale encoding.